controllers/apps/transformer_component_status.go

```diff
@@ -216,15 +217,15 @@ func (t *componentStatusTransformer) isAllConfigSynced(transCtx *componentTransf
 
 	configurationKey := client.ObjectKey{
```

Comment:
This condition for component status should be removed, since there are no more custom resources to maintain for config/script templates.